scripts/quests/otherAreas/A_Generous_General.lua

```diff
@@ -0,0 +1,290 @@
+-----------------------------------
+-- A Generous General
+-----------------------------------
+-- Log ID: 4, Quest ID: 109
+-----------------------------------
+-- Oldton zone !pos 566 -12 685 106
+-- Faulpie     !gotoid 17719379
+-- iron box    !pos -141 7 157 11
+-----------------------------------
+local oldtonID = zones[xi.zone.OLDTON_MOVALPOLOS]
+local southernSanDoriaID = zones[xi.zone.SOUTHERN_SAN_DORIA]
+-----------------------------------
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.A_GENEROUS_GENERAL)
+
+local spawnedMobs =
+{
+    oldtonID.mob.GENEROUS_GENERAL_OFFSET,
+    oldtonID.mob.GENEROUS_GENERAL_OFFSET + 1,
+    oldtonID.mob.GENEROUS_GENERAL_OFFSET + 2,
+    oldtonID.mob.GENEROUS_GENERAL_OFFSET + 3,
+    oldtonID.mob.GENEROUS_GENERAL_OFFSET + 4,
+    oldtonID.mob.GENEROUS_GENERAL_OFFSET + 5,
+    oldtonID.mob.GENEROUS_GENERAL_OFFSET + 6,
+}
+
+quest.reward =
+{
+    item  = xi.item.CHOPLIXS_COIF,
+    title = xi.title.MOBLIN_KINSMAN,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+                player:getQuestStatus(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.AN_AFFABLE_ADAMANTKING) ~= xi.questStatus.QUEST_ACCEPTED and
+                player:getQuestStatus(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.AN_UNDERSTANDING_OVERLORD) ~= xi.questStatus.QUEST_ACCEPTED and
+                player:getQuestStatus(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.A_MORAL_MANIFEST) ~= xi.questStatus.QUEST_ACCEPTED and
+                player:getVar('BstHeadGearQuest_Conquest') < NextConquestTally() and
```

Comment:
If this variable is unique to this quest, then it should follow standard quest vars, at absolute minimum, it should be an expiring charvar with a check against 0

Reply:
All of the Beast headgear quests share the same time block of conquest tally, so if you start one headgear quest you can't start another unless you cancel the one you're currently on. Trying to find the source I saw that bit of info in though..

Reply:
(unless the quest is cancelled)

Reply:
Can't find the source I had seen this in so moving this over to a quest var